File: Raven.Abstractions/Data/BackupRequest.cs
//-----------------------------------------------------------------------
// <copyright file="BackupRequest.cs" company="Hibernating Rhinos LTD">
//     Copyright (c) Hibernating Rhinos LTD. All rights reserved.
// </copyright>
//-----------------------------------------------------------------------
namespace Raven.Abstractions.Data
{
	public class BackupRequest
	{
		public string BackupLocation { get; set; }
		public DatabaseDocument DatabaseDocument { get; set; }
	}
}

File: Raven.Client.Java/src/main/java/net/ravendb/abstractions/data/BackupRequest.java
package net.ravendb.abstractions.data;


public class BackupRequest {
  private String backupLocation;

  private DatabaseDocument databaseDocument;

  public String getBackupLocation() {
    return backupLocation;
  }

  public void setBackupLocation(String backupLocation) {
    this.backupLocation = backupLocation;
  }

  public DatabaseDocument getDatabaseDocument() {
    return databaseDocument;
  }

  public void setDatabaseDocument(DatabaseDocument databaseDocument) {
    this.databaseDocument = databaseDocument;
  }

}
